.github/workflows/pull-request.yml

```diff
@@ -150,4 +151,24 @@ jobs:
       run: dotnet build ${{ github.workspace }}/src/MIDebugEngine-Unix.sln
 
     - run: |
-        ${{ github.workspace }}/PublishOpenDebugAD7.sh -c Debug -o ${{ github.workspace }}/bin/DebugAdapterProtocolTests/Debug/extension/debugAdapters
\ No newline at end of file
+        ${{ github.workspace }}/PublishOpenDebugAD7.sh -c Debug -o ${{ github.workspace }}/bin/DebugAdapterProtocolTests/Debug/extension/debugAdapters
+        cp ${{ github.workspace }}/bin/DebugAdapterProtocolTests/Debug/CppTests/TestConfigurations/config_lldb.xml ${{ github.workspace }}/bin/DebugAdapterProtocolTests/Debug/CppTests/config.xml
+
+    - name: Download LLDB-MI
+      run: |
+        # Download the latest version of lldb-ui that vscode-cpptools uses.
+        curl https://download.visualstudio.microsoft.com/download/pr/173e6ced-0717-401d-87fc-169ca3424c72/f1228fd847c140b7f9839612f497bb7a/lldb-mi-10.0.0.zip --output ${{ github.workspace }}/lldb-mi-10.0.0.zip
+        unzip ${{ github.workspace }}/lldb-mi-10.0.0.zip
+
+        # Ensure we can run it or we will get permission denied.
+        sudo chmod 755 /Users/runner/work/MIEngine/MIEngine/debugAdapters/lldb-mi/bin/lldb-mi
```

Comment:
Similarly, would it make sense to drop this in a .sh script so it would be easier to setup for devs?